spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -31,9 +39,35 @@ case class ColumnName(columnName: String, alias: Option[String] = None) extends
   override def selectedAs = alias.getOrElse(columnName)
   override def toString: String = columnName
 
+  def overwrite() = CollectionColumnName(columnName, alias, CollectionOverwrite)
+  def add() = CollectionColumnName(columnName, alias, CollectionAppend)
+  def append() = CollectionColumnName(columnName, alias, CollectionAppend)
+  def prepend() = CollectionColumnName(columnName, alias, CollectionPrepend)
+  def remove() = CollectionColumnName(columnName, alias, CollectionRemove)
```

Comment:
What about calling these methods:
`forOverwriting`, `forAdding`, `forAppending`, `forPrepending`, `forRemoving`.
This would read almost like "select column1 for appending". What do you think?